### ISSUE TITLE
Someone STOLE your code

### ISSUE THREAD

**Issue by professionalNoob123** [BhavyaCodes/covid-tracker-material-ui-react#39]
Hey @BhavyaCodes ,
I am here to just let you know that the code for this repo, you have worked hard on, has been stolen by a teenager in India (I believe South India). @Jaideep25 stole your code and uploaded it to his repo, and then uploaded it to his website hosted by Vercel.

- Repo: https://github.com/Jaideep25/Covid-tracker
- Website: https://covid-tracker-jaideep25.vercel.app/

I believe this is a copyright violation since he doesn't follow the MIT license you issued (https://github.com/BhavyaCodes/covid-tracker-material-ui-react/blob/master/LICENSE).

I'm notifying you because of this violation. And just to let you know that your work matters.

Thanks,
@professionalNoob123

**Comment by BhavyaCodes** [comment]
Thanks for the heads up @professionalNoob123 , I can't access the repository and the website at this time, seems unavailable. Perhaps it was deleted. Yes i worked very hard on it so appreciate your message a lot but unfortunately this has happened before also, people would upload repo after removing all my commits and changing the name also in MIT licence file.

Message for people who want to work on this project - Please fork the repository and make any changes you like except modifying the license and dont claim that it's your work. **Cheap shortcuts will not help you with anything.**
Thank you 🙏

**Comment by professionalNoob123** [comment]
> I can't access the repository and the website at this time, seems unavailable. Perhaps it was deleted

Yes, he **_DELETED_** it. @Jaideep25 must have gotten a notification (since I mentioned him), and then he deleted the repo + the website.

Probably he was ashamed of himself and didn't even bother to comment in this issue, (maybe attitude, who knows?). 

Moreover, several months back (I believe December 7, 2022), I saw him copy of someone's portfolio (literally everything except the original author's name) and deployed it from Vercel (to https://jaii25.vercel.app/ , **THIS LINK IS NOW UNAVAILABLE**)... he cannot even express himself from his own words and would rather violate copyright. 

PS: And guess what, on that website he called himself "a UI/UX dev, who helps to turn ideas into accessible experiences." 

> Cheap shortcuts will not help you with anything.

Agreed, I hope @jaideep25 is able to understand the true value of hard work.

Thank You
